api/go/info/info.pb.go

```diff
@@ -804,6 +804,116 @@ func (ZDeviceState) EnumDescriptor() ([]byte, []int) {
 	return file_info_info_proto_rawDescGZIP(), []int{11}
 }
 
+type StorageRaidType int32
+
+const (
+	StorageRaidType_STORAGE_RAID_TYPE_UNSPECIFIED StorageRaidType = 0
+	StorageRaidType_STORAGE_RAID_TYPE_RAID_ZERO   StorageRaidType = 1 //RAID-0
+	StorageRaidType_STORAGE_RAID_TYPE_RAID_MIRROR StorageRaidType = 2 // RAID-1
+	StorageRaidType_STORAGE_RAID_TYPE_RAIDZ       StorageRaidType = 3 // raidz (RAID-5)
```

Comment:
raidz is a generic term for all raid levels. This should be raidz1.

Reply:
corrected